specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json

```diff
@@ -0,0 +1,548 @@
+       "$ref": "#/definitions/BatchInputV2"
+      }
+     },
+     {
+      "name": "numberOfLanguagesToDetect",
+      "in": "query",
+      "description": "(Optional. Deprecated) Number of languages to detect. Set to 1 by default. Irrespective of the value, the language with the highest score is returned.",
+      "required": false,
+      "type": "integer",
+      "format": "int32"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call results in the detected language with the highest probability for each valid document",
+      "schema": {
+       "$ref": "#/definitions/LanguageBatchResultV2"
+      },
+      "examples": {
+       "application/json": {
+        "documents": [
+         {
+          "id": "1",
+          "detectedLanguages": [
+           {
+            "name": "English",
+            "iso6391Name": "en",
+            "score": 1
+           }
+          ]
+         },
+         {
+          "id": "2",
+          "detectedLanguages": [
+           {
+            "name": "French",
+            "iso6391Name": "fr",
+            "score": 1
+           }
+          ]
+         },
+         {
+          "id": "3",
+          "detectedLanguages": [
+           {
+            "name": "Spanish",
+            "iso6391Name": "es",
+            "score": 1
+           }
+          ]
+         },
+         {
+          "id": "4",
+          "detectedLanguages": [
+           {
+            "name": "(Unknown)",
+            "iso6391Name": "(Unknown)",
+            "score": 0
+           }
+          ]
+         }
+        ],
+        "errors": null
+       }
+      }
+     },
+     "default": {
+      "description": "Error Response",
+      "schema": {
+       "$ref": "#/definitions/ErrorResponse"
+      }
+     }
+    },
+    "deprecated": false
+   }
+  },
+  "/v2.0/sentiment": {
+   "post": {
+    "tags": [
+     "Sentiment"
+    ],
+    "summary": "The API returns a numeric score between 0 and 1. \r\n            Scores close to 1 indicate positive sentiment, while scores close to 0 indicate negative sentiment. \r\n            Sentiment score is generated using classification techniques. \r\n            The input features to the classifier include n-grams, features generated from part-of-speech tags, and word embeddings. \r\n            See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview#supported-languages\">Text Analytics Documentation</a> for details about the languages that are supported by sentiment analysis.",
+    "operationId": "Sentiment",
+    "consumes": [
+     "application/json"
+    ],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     },
+     {
+      "name": "input",
+      "in": "body",
+      "description": "Collection of documents to analyze.",
+      "required": true,
+      "schema": {
+       "$ref": "#/definitions/MultiLanguageBatchInputV2"
```

Comment:
Can this just be "MultiLanguageBatchInput"? The package namespace should differ so if there are same names they should be able to coexist.

Reply:
The Swagger spec was generated from Swashbuckle and that's the name of the type used. There will likely be an update later on to remove this.

Reply:
Just be aware an update later will be a breaking change for you. Is that going to be okay for you?

Reply:
Yeah, the next change to remove this will likely be breaking change in of itself, including changing schema. We will take type names into concern then. THanks!